app/vmctl/vm_native.go

```diff
@@ -111,6 +112,9 @@ func (p *vmNativeProcessor) do(ctx context.Context, f native.Filter, srcURL, dst
 }
 
 func (p *vmNativeProcessor) runSingle(ctx context.Context, f native.Filter, srcURL, dstURL string, bar *pb.ProgressBar) error {
+	if p.isSilent {
+		fmt.Printf("Continue import process with filter %s:\n", f.String())
```

Comment:
Is it better to use log.Println to make it more consistent?

Reply:
I will make changes like you propose in the last comment